stan-ballerina/Package.md

```diff
@@ -1,16 +1,15 @@
 ## Package Overview
 
-The `stan` library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io
-/"> Ballerina </a> language.
+The `stan` library is one of the standard library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
 
 This package provides the capability to send and receive messages by connecting to the NATS streaming server. NATS Streaming is a data streaming system powered by NATS that adds persistence, message replay, durable subscriptions, etc. to NATS.
 
 ### Report Issues
 
-To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina Standard Library parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
+To report bugs, request new features, start new discussions, view project boards, etc., go to the <a target="_blank" href="https://github.com/ballerina-platform/ballerina-standard-library">Ballerina Standard Library parent repository</a>.
 
```

Comment:
Here let's make the first letter lower case. 

Ballerina standard library instead of Ballerina Standard Library

Reply:
Fixed in fe68e691cfd78611f170dce0d02a638d235c4048